src/converters/piral-blazor/src/converter.ts

```diff
@@ -1,6 +1,27 @@
 import type { BaseComponentProps, ForeignComponent } from 'piral-core';
 import { addGlobalEventListeners, attachEvents, removeGlobalEventListeners } from './events';
 import { activate, deactivate, createBootLoader } from './interop';
+import { BlazorOptions } from './types';
+
+function prefixMediaSources(component: Element, prefix: string) {
+  const prefixAttributeValue = (el, attr) => el.setAttribute(attr, prefix + el.getAttribute(attr));
+
+  const ruleSet = [
```

Comment:
This can be made a module global variable - thus not having a reallocation every time the function is called.